### PR TITLE
start a README section for editor integration links

### PR DESCRIPTION
**Pull request by ef4**
I think this is a good place to link people to details on editor integrations that make it more practical to try these new syntaxes.

Thanks @nullvoxpopuli for pointing out vscode-glimmer, and I know you also may be able to add an entry here for neovim.

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
related comment (lost to the abyss?) https://github.com/ef4/ember-template-imports/commit/314a1003311a80cab46276c78b8e34e1feb99ce1#commitcomment-59366420

File: README.md [hunk 1]
```diff
@@ -283,6 +283,24 @@ Installation
...
+Editor Integrations
+------------------------------------------------------------------------------
+
+To get syntax highlighting inside embedded templates and support for the GJS
+file extension, you may need to configure your editor.
+
+### Visual Studio Code
+The [vscode-glimmer](https://marketplace.visualstudio.com/items?itemName=chiragpat.vscode-glimmer) plugin handles syntax highlighting for both proposed formats.
+
+### Neovim
```
Comment (NullVoxPopuli) [inline README.md:295]: 💯

File: README.md [hunk 2]
```diff
@@ -283,6 +283,24 @@ Installation
...
 ```
 
+Editor Integrations
+------------------------------------------------------------------------------
+
+To get syntax highlighting inside embedded templates and support for the GJS
+file extension, you may need to configure your editor.
+
+### Visual Studio Code
+The [vscode-glimmer](https://marketplace.visualstudio.com/items?itemName=chiragpat.vscode-glimmer) plugin handles syntax highlighting for both proposed formats.
```
Comment (NullVoxPopuli) [inline README.md:293]: 💯

**Review by NullVoxPopuli — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -283,6 +283,24 @@ Installation
...
+
+### Neovim
+
+[Example Neovim Config](https://github.com/NullVoxPopuli/dotfiles/blob/master/home/.config/nvim/lua/plugins.lua#L69) with support for good highlighting of embedded templates in JS and TS, using:
+
+- https://github.com/nvim-treesitter/nvim-treesitter
+- https://github.com/alexlafroscia/tree-sitter-glimmer
+
+
+
```
Comment (NullVoxPopuli) [inline README.md:303]: I'm not quite sure how to make (neo)vim detect filetypes by default yet, 
but, I think we'd want something like
```vim
  autocmd BufNewFile,BufRead *.gts setfiletype typescript.glimmer
  autocmd BufNewFile,BufRead *.gjs setfiletype javascript.glimmer
```

example: https://github.com/NullVoxPopuli/dotfiles/blob/master/home/.config/nvim/init.vim#L20-L25

cc @alexlafroscia

**Comment by ef4** [comment]
With the RFC in final comment period I expect growing interest in people trying to do this. Can we merge these docs changes? Even imperfect is better than nothing.

**Comment by chriskrycho** [comment]
Doh, I didn’t remember I had merge rights. Yes, very much agreed re: something-over-nothing, so merging now!

File: README.md (hunk 3 above)
Comment (chriskrycho) [reply README.md:303]: @NullVoxPopuli mind figuring that out and opening a PR to add it once you do? Thanks!

Comment (chriskrycho) [reply README.md:303]: Excellent!

**Comment by NullVoxPopuli** [comment]
both vim and neovim will have this identification _soonish_: https://github.com/vim/vim/commit/cdf717283ca70b18f20b8a2cefe7957083280c6f
- javascript.glimmer
- typescript.glimmer

🎉

**Pull request merged**